recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+MACOSX_SDK_VERSION:            # [osx and x86_64]
```

Comment:
Why was this added?

Reply:
Removed and rerendered. This was due to local testing differences. Apologies for the noise.